src/v/kafka/server/group.cc

```diff
@@ -2878,14 +2878,43 @@ ss::future<> group::do_try_abort_old_tx(model::producer_identity pid) {
         }
     }
 
-    std::optional<model::tx_seq> tx_seq;
-
     auto p_it = _prepared_txs.find(pid);
     if (p_it != _prepared_txs.end()) {
-        tx_seq = p_it->second.tx_seq;
-    }
+        auto r = co_await _tx_frontend.local().try_abort(
+          model::partition_id(0), // Partition_id 0???
```

Comment:
> kafka: Add logix to expire old tx

would it make sense to combine this commit with an older commit? at least please explain what the commit is doing. in many ways the commit message `Add logix to expire old tx` could describe the entire PR, but it shoudl be describing what is happening in this commit.